src/dashboard/trade/placeOrder/AmountDetails.tsx

```diff
@@ -21,12 +21,72 @@ type AmountDetailsProps = {
 const useStyles = makeStyles(() => {
   return createStyles({
     iconContainer: {
+      color: "#979797",
       display: "flex",
       alignItems: "center",
+      fontSize: "16px",
+    },
+    text: {
+      color: "#979797",
+      fontSize: "16px",
+      padding: "2px 0",
+    },
+    textHighlighted: {
+      color: "#f2f2f2",
+    },
+    toolTip: {
+      color: "#f2f2f2",
+      backgroundColor: "#0c0c0c",
+      fontSize: "13px",
     },
   });
 });
 
+const CustomSlider = withStyles({
+  root: {
+    padding: "8px 0",
+  },
+  thumb: {
```

Comment:
The thumb looks a bit off (it's downwards from the line + the white border is not continuous).